### ISSUE TITLE
Task: Render the set site logo and description from user input. 

### ISSUE THREAD

**Issue by alshakero** [Automattic/wp-calypso#67365]
### Details

**Note**: there is a small chance, this is already done by [this](https://github.com/Automattic/wp-calypso/pull/67250), please verify before starting.

The site is already configured, but the applied patterns don’t reflect that. Conversation [here](https://href.li/?p1661499387489309-slack-C02M88KJ684).

p1661499387489309-slack-C02M88KJ684

### Checklist

- [ ] When done, the pattern's logo block should have the uploaded logo

**Comment by heavyweight** [comment]
It seems that everything that this needs to work is in place.

The selected image is shown in the lunchpad
<img width="1151" alt="Screenshot 2022-09-07 at 20 13 13" src="https://user-images.githubusercontent.com/7000684/188949529-125612fe-71a8-4141-88a5-af2a8ee0bdb5.png">

The selected image is applied to the site
<img width="1156" alt="Screenshot 2022-09-07 at 20 13 27" src="https://user-images.githubusercontent.com/7000684/188949561-00d5952a-1ee1-45c9-9b29-52748e1526ee.png">

**Comment by alshakero** [comment]
That's correct! Forgot to close this one.
